Formula/gnupg@1.4.rb

```diff
@@ -0,0 +1,48 @@
+class GnupgAT14 < Formula
+  desc "GNU Pretty Good Privacy (PGP) package"
+  homepage "https://www.gnupg.org/"
+  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-1.4.21.tar.bz2"
+  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/gnupg/gnupg-1.4.21.tar.bz2"
+  sha256 "6b47a3100c857dcab3c60e6152e56a997f2c7862c1b8b2b25adf3884a1ae2276"
+
+  keg_only :versioned_formula
+
+  depends_on "curl" if MacOS.version <= :mavericks
+  depends_on "libusb-compat" => :optional
+
+  def install
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --disable-asm
+    ]
+    args << "--with-libusb=no" if build.without? "libusb-compat"
+
+    system "./configure", *args
+    system "make"
+    system "make", "check"
+
+    # we need to create these directories because the install target has the
+    # dependency order wrong
+    [bin, libexec/"gnupg"].each(&:mkpath)
+    system "make", "install"
```

Comment:
I'm not actually against that idea, where the system remains semi-similar to how it is now, except people are shunted across to `gnupg` (2.1.x) which becomes what `gnupg2` currently is in this system.

That would avoid significant rewrites of the `gpg` stuff in `brew`, but I'm mostly wary that 1.4.x remains quite popular because of how portable it is. On a modern system you require zero dependencies at all.